docs/functionalities/biology/Biology_summary/Custom_entity/stats_summary.csv

```diff
@@ -0,0 +1,3 @@
+LOINC_concept_code,AnaBio_concept_code,LOINC_concept_name,AnaBio_concept_name,unit_source_value,count,mean,std,min,25%,50%,75%,max,MAD,max_threshold,min_threshold
```

Comment:
Are these csv and html supposed to be there?

Reply:
Yep, they come from fake data and were erroneously removed !